### ISSUE TITLE
reduce color palette of resulting png images

### ISSUE THREAD

**Issue by magiruuvelvet** [magiruuvelvet/jimaku-editor#3]
3rd party software is required at the moment before the pgs-encoder accepts the image files, reduce the color palette and depth of the png to `8-bit colormap` natively in the renderer

**current behavior**
```
test1.png: PNG image data, 355 x 62, 8-bit gray+alpha, non-interlaced
```

**expected behavior**
```
test1.png: PNG image data, 355 x 62, 8-bit colormap, non-interlaced
```

**Comment by magiruuvelvet** [comment]
workaround implemented with command support

**Comment by magiruuvelvet** [comment]
fixed in 6ed9f5d3f9511a5c42c320d4a6306179e029f6d2

the custom command support was originally implemented as workaround for this, but i'm keeping it as it is useful for automated post processing of subtitle images
